.github/actions/test-dotnet/run-tests.ps1

```diff
@@ -12,23 +10,19 @@ function Configure-Environment($hash) { Set-Item @hash }
 
 if ($Env:RUNNER_OS -eq "Windows")
 {
-    Configure-Environment @{
-        Path = 'Env:\Lombiq_Tests_UI__SqlServerDatabaseConfiguration__ConnectionStringTemplate'
-        Value = 'Server=.\SQLEXPRESS;Database=LombiqUITestingToolbox_{{id}};Integrated Security=True;MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5'
-    }
+    $Env:Lombiq_Tests_UI__SqlServerDatabaseConfiguration__ConnectionStringTemplate =
+        'Server=.\SQLEXPRESS;Database=LombiqUITestingToolbox_{{id}};Integrated Security=True;MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5'
```

Comment:
I know this was mixed up before, but please use double quotes for strings in any new code.